### PR TITLE
short alias to set/show namespace

### PR DESCRIPTION
**Pull request by mtilson**
usage:
```
$ kn # shows current namespace
$ kn default # switch to specified namespace
```

### CODE REVIEW

**Comment by divya-mohan0209** [comment]
Hello @mtilson , please sign the CLA

**Comment by jimangel** [comment]
Hey there! @mtilson, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

**Review by Arhell — COMMENTED** [review]
/check-cla

**Comment by mtilson** [comment]
I have signed the CLA just now.

**Comment by mtilson** [comment]
JFI: It seems like @krousey is not in the list of `sig-docs-en-reviews` in the [owners aliases](https://github.com/kubernetes/website/blob/main/OWNERS_ALIASES), which should be according to [content/en/OWNERS](https://github.com/kubernetes/website/blob/main/content/en/OWNERS) and [OWNERS](https://github.com/kubernetes/website/blob/main/OWNERS)

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ kubectl config set-context gce --user=cluster-admin --namespace=foo \
   && kubectl config use-context gce
 
 kubectl config unset users.foo                       # delete user foo
+
+# short alias to set/show context/namespace (useful for CKAD/CKA exams)
```
Comment (sftim) [inline content/en/docs/reference/kubectl/cheatsheet.md:77]: It's be better to call out that this only works in `bash` and compatible shells. You also have to have a current context for the error handling to work here:
```
$ kn no-such-namespace
error: no current context is set
error: current-context must exist in order to minify
$ 
```
---

Despite the term “cheat sheet” in the title, the page is mainly aimed at Kubernetes end users and exam candidates.
```suggestion
# short alias to set/show context/namespace
```

@mtilson I don't _object_ to this PR merging, but I'm also not inclined to approve it. Keeping the cheat sheet page concise is useful for our end user readers who aren't sitting exams.

Comment (mtilson) [reply content/en/docs/reference/kubectl/cheatsheet.md]: No problem, the main idea was to have, probably, the shortest way to setup something  (aliases in this case) in `shell` to juggle with contexts and namespaces. It is especially useful during exams, as there is no time to setup heavier plugins like `kubectx` or `kubens`, but it also could be useful for general end user who looks through this cheatsheet. And yes, as `bash` is the shell used during exam I tried to use its constructions, but it seems it is possible to convert them to POSIX compatible ones.

**Comment by kbhawkey** [comment]
Thanks @mtilson .
/approve

**Comment by sftim** [comment]
Thanks

/lgtm

**Pull request merged**